### PR TITLE
Add acronyms

### PR DESCRIPTION
**Pull request by haszi**
Add acronyms and corresponding entities for frequently used abbreviations. This is mostly based on abbreviations found in the cURL chapter of the docs.

### CODE REVIEW

File: entities/acronyms.xml [hunk 1]
```diff
@@ -282,6 +360,12 @@
     <simpara>Privacy-Enhanced Mail</simpara>
    </listitem>
   </varlistentry>
+  <varlistentry>
+   <term>Phar</term>
```
Comment (cmb69) [inline entities/acronyms.xml:364]: Not sure if "Phar" is considered an acronym, but it's okay for me to add it.

File: entities/global.ent [hunk 2]
```diff
@@ -606,5 +606,653 @@
...
+<!ENTITY yield '<link xmlns="http://docbook.org/ns/docbook" linkend="control-structures.yield"><literal>yield</literal></link>'>
+<!ENTITY yield.from '<link xmlns="http://docbook.org/ns/docbook" linkend="control-structures.yield.from"><literal>yield from</literal></link>'>
+
+<!ENTITY callback.cmp '<methodsynopsis xmlns="http://docbook.org/ns/docbook"><type>int</type><methodname><replaceable>callback</replaceable></methodname><methodparam><type>mixed</type><parameter>a</parameter></methodparam><methodparam><type>mixed</type><parameter>b</parameter></methodparam></methodsynopsis>'>
+
+<!ENTITY php.ini '<filename xmlns="http://docbook.org/ns/docbook">php.ini</filename>'>
+<!ENTITY user-ini '<filename xmlns="http://docbook.org/ns/docbook">.user.ini</filename>'>
+<!ENTITY httpd.conf '<filename xmlns="http://docbook.org/ns/docbook">httpd.conf</filename>'>
+<!ENTITY htaccess '<filename xmlns="http://docbook.org/ns/docbook">.htaccess</filename>'>
+
```
Comment (cmb69) [inline entities/global.ent:1216]: This looks wrong (duplication?)

**Review by cmb69 — COMMENTED** [review]
Thank you very much for woking on this!

File: entities/global.ent [hunk 3]
```diff
@@ -606,5 +606,653 @@
...
+<!ENTITY spec.http1.1  'http://tools.ietf.org/html/rfc7231'>
+<!ENTITY spec.soap1.1  'http://www.w3.org/TR/soap11/'>
+<!ENTITY spec.soap1.2  'http://www.w3.org/TR/soap12/'>
+<!ENTITY spec.wsdl     'http://www.w3.org/TR/wsdl'>
+
+<!ENTITY book.applied.cryptography 'Applied Cryptography by Schneier (ISBN 0-471-11709-9)'>
+<!ENTITY book.programming.classics "Programming Classics: Implementing the World's Best Algorithms by Oliver (ISBN 0-131-00413-1)">
+
+<!ENTITY email.harald                'harald.radi@nme.at'>
```
Comment (Girgias) [inline entities/global.ent:1153]: I'm not sure this is a good idea?

File: entities/global.ent [hunk 4]
```diff
@@ -606,5 +606,653 @@
...
+<!ENTITY email.php.install           'php-install@lists.php.net'>
+<!ENTITY email.php.install.subscribe 'php-install+subscribe@lists.php.net'>
+<!ENTITY email.php.doc               'phpdoc@lists.php.net'>
+<!ENTITY email.php.doc.license       'doc-license@lists.php.net'>
+<!ENTITY email.php.doc.subscribe     'phpdoc+subscribe@lists.php.net'>
+<!ENTITY email.sample                'yourname@example.com'>
+
+<!ENTITY link.superglobals '<link xmlns="http://docbook.org/ns/docbook" linkend="language.variables.predefined">superglobals</link>'>
+<!ENTITY link.autoload '<link xmlns="http://docbook.org/ns/docbook" linkend="language.oop5.autoload">__autoload</link>'>
```
Comment (Girgias) [inline entities/global.ent:1162]: I'm not sure about this one, as those should probably be fixed to redirect to the SPL version of autoloading.

File: entities/global.ent (hunk 2 above)
Comment (Girgias) [reply entities/global.ent:1216]: Seems like it

Comment (haszi) [reply entities/global.ent:1216]: Ooops, what did I do?? I'm on it.

File: entities/global.ent (hunk 3 above)
Comment (haszi) [reply entities/global.ent]: This was already part of the global entities file, I just accidentally duplicated all but the acronyms part of it.

File: entities/global.ent (hunk 4 above)
Comment (haszi) [reply entities/global.ent]: This is also already part of the global entities file.

File: entities/global.ent (hunk 3 above)
Comment (cmb69) [reply entities/global.ent]: @Girgias, heh

https://github.com/php/doc-en/blob/72605fc9de80bd30f50b112e447326dfe6c2a523/faq/com.xml#L227-L228

Comment (Girgias) [reply entities/global.ent]: I wonder if they would even reply :')

**Review by Girgias — APPROVED** [review]
Thank you!

**Pull request merged**